experimental/addons/process-exporter/README.md

```diff
@@ -0,0 +1,53 @@
+# process-exporter
```

Comment:
why do we add a process-exporter in this PR?

Reply:
I don't know why this is happening too. This plugin already exists.

Reply:
Done